erpnext/patches/v13_0/update_returned_qty_in_pr_dn.py

```diff
@@ -13,7 +13,7 @@ def execute():
 	frappe.reload_doc('stock', 'doctype', 'stock_settings')
 
 	def update_from_return_docs(doctype):
-		for return_doc in frappe.get_all(doctype, filters={'is_return' : 1, 'docstatus' : 1}):
+		for return_doc in frappe.get_all(doctype, filters={'is_return' : 1, 'docstatus' : 1, 'return_against': 'IS NOT NULL'}):
```

Comment:
```suggestion
		for return_doc in frappe.get_all(doctype, filters={'is_return' : 1, 'docstatus' : 1, 'return_against': ('!=', '')}):
```

In filters `{key: value}` means `key = value` in SQL. 

You can confirm by adding `debug=True` to inspect generated query.

Reply:
Good to know! Thanks!